ddtrace/appsec/processor.py

```diff
@@ -190,6 +192,57 @@ def _update_rules(self, new_rules):
         except TypeError:
             log.debug("Error updating ASM rules", exc_info=True)
 
+    def _set_metrics(self):
```

Comment:
We should add test coverage for test metrics. Right now there is no regression tests for appsec processor metrics, the values can change/break at any point and we won't be notified.

Reply:
Thank you so much @mabdinur , Nice idea to add performance tests :+1: 
About integration and regression tests is it enough with these tests?
- https://github.com/DataDog/dd-trace-py/pull/5046/files#diff-e56afc084b0d6a7e0a5d1c16ead0bc8d9916c99127c9c1d2224130b9e940e11eR1
- https://github.com/DataDog/dd-trace-py/pull/5046/files#diff-c210bd10d4e51b5ab136feb9a72477acc700d4183c3c59da7b942fc0de6d5d53R1
- https://github.com/DataDog/system-tests/pull/890

DM me if you want to share your thoughts and discuss the best approach :upside_down_face: